src/cloud/project/new-relic.md

```diff
@@ -248,9 +248,9 @@ The following example shows a New Relic Insights dashboard created from queries
 
 For further information and examples, see [Introduction to New Relic Logs][New Relic Logs] and [Introduction to New Relic Insights][New Relic Insights].
 
-### Monitor performance with alert policies
+### Monitor performance with the Managed Alerts for Magento alert policy
```

Comment:
Whoa, I don't recommend a heading this long, because it looks bad in the page navigation.

```suggestion
### Monitor performance with Managed Alerts
```